content/plugins/hashed-module-ids-plugin.md

```diff
@@ -0,0 +1,11 @@
+---
+title: HashedModuleIdsPlugin
+contributors:
+  - shaodahong
+---
+
+This plugin will cause hashes to be based on the relative path of the module, generating a four character string as the module id. Suggested for use in production.
+
+``` js
+new webpack.HashedModuleIdsPlugin()
```

Comment:
Please also document the available options: https://github.com/webpack/webpack/blob/master/lib/HashedModuleIdsPlugin.js#L9-L15

Some options are already documented here: https://webpack.js.org/configuration/output/#output-hashdigest